### DISCUSSION TITLE
Accessibility Statement says the non accessible aspects of the app will be improved as soon as possible - is this true?

### DISCUSSION THREAD

**Discussion by Ein-Tim** [corona-warn-app/cwa-website#3284] [Q&A] [answered]
## Your Question
* Website URL Path: https://www.coronawarn.app/en/accessibility/
* Line / Paragraph: Level of compliance with requirements 
* Question: The site lists the things where the app is not (well) accessible. It says that you are working to resolve these issues as soon as possible. As the accessibility page was first published on March 8th, I wonder what "as soon as possible" means. The page was first linked to from version 2.17 and we are now on version 2.21 but still non of the issues mentioned on the page were addressed. If you are not working on resolving the issues, please remove the statement. I offer a PR for this.

**Comment by dsarkar** [comment]
@Ein-Tim Good morning, I will forward this, however, I see little chance to get approval for this text change. ASAP is a very vague term. Obviously, not everything can be done at the same time, and there is certainly some sort of prioritising/selecting of working tickets, i.e. some other higher prioritised  bugs or features will be worked on before these issues can be addressed. In that sense ASAP is not a wrong therm, I dare to say. Anyway, I will check if can change the wording. Enjoy your Sunday, DS

**Comment by Ein-Tim** [comment]
Good morning @dsarkar!

Thank you for forwarding it, I guess the statement came right from the legal department so yes, it is quite difficult to get an approval for this.
Maybe, you could just check with them or the team if there is actually already a ticket created for the stuff and maybe developers are already assigned to it.

Enjoy your Sunday!

**Comment by dsarkar** [comment]
@Ein-Tim Indeed there is an own epic for "Accessibility" and there is already a number internal tickets documenting issues regarding accessibility.

**Comment by Ein-Tim** [comment]
@dsarkar Okay, thanks for the feedback! I'm closing this issue now.

**Comment by Ein-Tim** [comment]
I'm reopening this issue now (5 months later) and pinging @GisoSchroederSAP & @sapcoder123 to ask for an update. Non of the issues your own accessibility assessment were found. Is this not a priority for the client or why was this not addressed?

**Comment by Ein-Tim** [comment, marked as answer]
I don't really know why this has been converted into a discussion thread, however, PR #3360 removes the phrases which state that the non-accessible aspects of the app & the website will be improved. While I think this is not the best solution, which would be to resolve the issues, I accept this change with the background of the CWA project ramp down.
